src/components/JoinUs/Learning.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Padding, Margin } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { Row, CompensatedRow, CompensatedCol } from '../grid'
+import Image from '../Common/Image'
+import { H2, Paragraph } from '../Typography'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColum,
+  Separator,
+  List,
+  Hr,
+  TitleAndBody
+} from './elements'
+
+const InsightAnchor = styled.a`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+`
+
+const Learning = ({ data: { title, text, featuredInsights, list } }) => (
+  <Section greyBg>
+    <Row>
+      <FirstColumn>
+        <H2>{title}</H2>
+      </FirstColumn>
+      <SecondColum width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>
+        <List list={list} />
+      </SecondColum>
+    </Row>
+    <Separator />
+    <TitleAndBody
+      title="Some insights into personal growth at YLD"
+      body={text}
+    />
+    <CompensatedRow>
+      {featuredInsights.map((el, idx) => (
+        <CompensatedCol width={[1, 1, 1, 4 / 12]} key={idx}>
+          <InsightAnchor href={el.url}>
+            <Padding top={2} style={{ width: '100%' }}>
+              <Margin bottom={1}>
```

Comment:
How come we have margin bottom on the image & padding top on the paragraph? Do we need this one? Maybe I'm missing something.

Reply:
In order to get those 18px distance between them (as you mentioned above), this is being done by setting a 6px margin bottom on the image and a 16px padding top on the paragraph. I guess this can be applied to different elements, but the end result's the same. Not sure what you mean

Reply:
You're right, we have a measure of 18 px on our theme, it's much quicker to use that as a padding on the `p`. It's done now :)